resources/schema.py

```diff
@@ -61,6 +101,8 @@ class Meta:
 class PierNode(graphql_geojson.GeoJSONType):
     class Meta:
         model = Pier
+        geojson_field = "location"
```

Comment:
tiny nitpick: this distorts slightly the git history, as now you have to check the previous commit before this one to figure out why this change was done, and this change, I suppose, does not have any functional causes/requirements.